x/issuance/types/msg.go

```diff
@@ -34,52 +35,40 @@ func NewMsgIssueTokens(sender sdk.AccAddress, tokens sdk.Coin, receiver sdk.AccA
 func (msg MsgIssueTokens) Route() string { return RouterKey }
 
 // Type returns a human-readable string for the message, intended for utilization within tags.
-func (msg MsgIssueTokens) Type() string { return "issue_tokens" }
+func (msg MsgIssueTokens) Type() string { return TypeMsgIssueTokens }
 
 // ValidateBasic does a simple validation check that doesn't require access to state.
 func (msg MsgIssueTokens) ValidateBasic() error {
-	if msg.Sender.Empty() {
+	if len(msg.Sender) == 0 {
 		return sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, "sender address cannot be empty")
 	}
 	if msg.Tokens.IsZero() || !msg.Tokens.IsValid() {
 		return sdkerrors.Wrapf(sdkerrors.ErrInvalidCoins, "invalid tokens %s", msg.Tokens)
 	}
-	if msg.Receiver.Empty() {
+	if len(msg.Receiver) == 0 {
```

Comment:
Could you update address validation in `msg.ValidateBasic` methods to use `sdk.AccAddressFromBech32`. I added a note in the confluence doc. Now that addresses are strings there's more ways they can be invalid.